spec/services/participants/check_and_set_completion_date_spec.rb

```diff
@@ -4,13 +4,17 @@
 
 RSpec.describe Participants::CheckAndSetCompletionDate do
   let(:cohort) { Cohort.previous || create(:cohort, :previous) }
+  let(:lead_provider) { create(:lead_provider, name: "Ambition Institute") }
   let(:school) do
     NewSeeds::Scenarios::Schools::School
       .new
       .build
+      .with_partnership_in(cohort:, lead_provider:)
       .chosen_fip_and_partnered_in(cohort:)
       .school
   end
+  let(:lead_provider_1) { create(:lead_provider, name: "Capita") }
+  let(:school_1) { NewSeeds::Scenarios::Schools::School.new.build.with_partnership_in(cohort:, lead_provider: lead_provider_1).chosen_fip_and_partnered_in(cohort:) }
```

Comment:
`school_1` doesn't seem to be used, not needed maybe?

Reply:
Great spot tks 👍 😄